.travis.yml

```diff
@@ -25,7 +25,7 @@
 # [1] https://docs.travis-ci.com/user/caching/#build-phases
 # [2] https://docs.travis-ci.com/user/customizing-the-build#build-timeouts
 
-dist: xenial
+dist: bionic
```

Comment:
Why need to change this value?

Reply:
Because the unit tests was running in a docker container with the default image of `ubuntu:18.04`. You can verify it at `ci/test/00_setup_env.sh`.
```
export DOCKER_NAME_TAG=${DOCKER_NAME_TAG:-ubuntu:18.04}
```